src/sentry/api/urls.py

```diff
@@ -1315,6 +1316,23 @@
                     OrganizationRequestProjectCreation.as_view(),
                     name="sentry-api-0-organization-request-project-creation",
                 ),
+                url(
```

Comment:
couple notes here: 
- SCIM doesn't use appended slashes, so this is on purpose.
- I decided to put these routes under /api/organizations/.... seemed more consistent with the rest of our apis than putting them top level or something
- the `/v2` refers to the version of SCIM, which Okta/Azure recommend having in the route (in case v3 is ever released its not ambiguous)

Reply:
I wonder if our nginx mucks with `/`es 🤔

Reply:
i believe it does not!